CHANGELOG.md

```diff
@@ -3,6 +3,13 @@
 This file tracks all the changes made to the client.
 This allows parsers such as Dependabot to provide a clean overview in pull requests.
 
+## v0.7.1
+
+- added quotation fallback rates for EU
+- add support for optional `country` attribute of `customer` (allows VAT calculation for EU if Vertex client is set up properly)
+- update quotation validation rules to require either `state` (US) or `country` (other countries)
+- update README.md
+
```

Comment:
Please use the updated changelog format in master. See here for details. https://keepachangelog.com/en/1.0.0/ also this would be version 0.9.1 now too.

Reply:
I think you managed to submit the review just before I pushed the last changes where I fixed the conflicts and updated the changelog formatting - it should be just fine atm. GitHub is currently showing incorrect base version of this file => giving me more credit for CHANGELOG.md than I should have - maybe updating the base of PR twice will help but didn't want to remove your comments (as I think it does exactly that - not 100% though)